cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/SpaceDevelopersOperations.java

```diff
@@ -103,18 +105,23 @@ public void assignSpaceDeveloper(String username) throws CreationException {
      *
      * @param usernameList List of users to be removed as a space developer.
      * @throws InvalidOperationException if usernameList is null/empty or spaceId is invalid.
+     * @throws UpdateException if the user is not found.
      */
-    public void removeSpaceDeveloper(List<String> usernameList) throws InvalidOperationException {
+    public void removeSpaceDeveloper(List<String> usernameList) throws UpdateException, InvalidOperationException {
```

Comment:
This could be avoided when using RuntimeException as super class for UpdateException

Reply:
Removed.